LIC_FILES_CHKSUM.sha256

```diff
@@ -1,10 +1,13 @@
 # Apache-2.0 license.
 d0f406b04e7901e6b4076bdf5fd20f9d7f04fc41681069fd8954413ac6295688  LICENSE
+
+5e0df8c845c742e76f2f64d2d9ce1b7e74a2422fddbc577ae6a56319083de0bf  src/common/vendor/yaml-cpp/test/gtest-1.11.0/googlemock/scripts/generator/LICENSE
```

Comment:
IIRC we had trouble with empty lines on some platforms (was it Mac? I can't recall). Just stick a comment marker there, and it should be fine.

Reply:
hah! Good to know! Fixing